src/Reservations/docs/Invoke-AzReservationsAvailableReservationScope.md

```diff
@@ -0,0 +1,246 @@
+---
+external help file:
+Module Name: Az.Reservations
+online version: https://docs.microsoft.com/powershell/module/az.reservations/invoke-azreservationsavailablereservationscope
+schema: 2.0.0
+---
+
+# Invoke-AzReservationsAvailableReservationScope
```

Comment:
Is Invoke-AzReservation**_s_**AvailableReservationScope your expected name? Should it be Invoke-AzReservationAvailableReservationScope? Could you check?

Reply:
Good catch! I change the name of this command to `Get-AzReservationAvailableScopes`

Reply:
Could you name it as Get-AzReservationAvailableScope as we need to keep singularity for naming cmdlet.

Reply:
Changed the name to Get-AzReservationAvailableScope